README.md

```diff
@@ -60,3 +60,28 @@ connection there will also receive routed packets.
 Pull-requests are accepted on GitHub. Make sure to check coding style with the
 provided script in tools/checkpatch, check for memory leaks with valgrind and
 test on real hardware.
+
+### Samples ###
+
+#### Sender & receiver ####
+
+One can test mavlink-router by using `examples/sender.py` and
+`examples/receiver.py` to simulate traffic of mavlink messages.
+First script send mavlink *ping* messages to a target mavlink system-id, and
+second receives and respond them.
+For instance:
+
+    $ PYTHONPATH=<path-to-pymavlink> python examples/sender.py 127.0.0.1:3000 100 0
```

Comment:
it's better to remove this PYTHONPATH from here. It's advised to install pymavlink with:

`pip install pymavlink`
or 
`pip2 install pymavlink`